internal/services/managedredis/managed_redis_resource.go

```diff
@@ -754,3 +804,44 @@ func dbLen(v interface{}) int {
 	}
 	return 0
 }
+
+func expandPersistence(input []PersistenceModel) *databases.Persistence {
+	res := databases.Persistence{}
+	if len(input) == 0 {
+		return &res
+	}
+
+	p11Model := input[0]
+
+	switch p11Model.Method {
+	case validate.DbPersistenceMethodRDB:
+		res.RdbEnabled = pointer.To(true)
+		res.RdbFrequency = pointer.ToEnum[databases.RdbFrequency](p11Model.BackupFrequency)
+	case validate.DbPersistenceMethodAOF:
+		res.AofEnabled = pointer.To(true)
+		res.AofFrequency = pointer.ToEnum[databases.AofFrequency](p11Model.BackupFrequency)
+	}
```

Comment:
can we enable both of them? Even if the Portal shows it as single option, what's the API behavior?